<!DOCTYPE HTML>
<!--
	Read Only by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Hugo Talibart | Bioinformatician</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>
	<body class="is-preload">

		<!-- Header -->
			<section id="header">
				<!-- <annonce>Available for an engineer position <br/> as of 12/2022</annonce> -->
				<header>
					<span class="image avatar"><img src="images/avatar.jpg" alt="" /></span>
					<h1 id="logo"><a href="index.html">Hugo Talibart</a></h1>
					<p>
						Postdoctoral researcher <br/>
						<a href="https://3bio.polytech.ulb.be/en/bioinfo">3BIO-BioInfo</a> <br/>
						Université Libre de Bruxelles <br/>
						<br/>
						<br/>
					</p>
				</header>
				<nav id="nav">
					<ul>
						<li><a href="#overview" class="active">overview</a></li>
						<li><a href="#publications">publications</a></li>
						<li><a href="#misc">misc.</a></li>
						<li><a href="#bio">resume</a></li>
						<li><a href="#contact">contact</a></li>
					</ul>
				</nav>
				<footer>
					<ul class="icons">
						<li><a href="https://twitter.com/HTalibart" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
						<!--<li><a href="#" class="icon brands fa-github"><span class="label">Github</span></a></li>-->
						<li><a href="mailto:hugo.talibart@irisa.fr" class="icon solid fa-envelope"><span class="label">Email</span></a></li>
					</ul>
				</footer>
			</section>

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Main -->
					<div id="main">
							<section id="overview">
								<div class="image main" data-position="center">
									<img src="images/banner.jpg" alt="" />
								</div>
								<div class="container">
									<header class="major">
										<h3>Overview</h3>
									</header>
		<p>
			I am a <b>computer scientist</b> with a passion for <b>molecular biology</b>.

 <br/>
			
			<br/>
			
			I discovered the dazzling world of proteins during my PhD at Université de Rennes 1 in <a href="http://www.irisa.fr/dyliss"/>Dyliss</a> team,
                under the supervision of <a href="http://people.rennes.inria.fr/Francois.Coste/">François Coste</a>
                and <a href="http://people.rennes.inria.fr/Jacques.Nicolas/">Jacques Nicolas</a>, from november 2017 to february 2021.
                
                <br/>
			<br/>
                
								This <a href="manuscript_TALIBART_Hugo.pdf">thesis</a> focused on <b>protein sequence homology search</b> and <b>protein family modelling</b>, with the added novelty of taking into account <b> residue co-evolution </b>.
                	
                	<br/>
			<br/>
                	
                	As of today, homologous protein sequences are standardly modeled using profile Hidden Markov Models (pHMMs), statistical models that represent protein families with positional conservation information. Widely used packages such as HH-suite provide tools to perform sequence-HMM alignment and HMM-HMM alignment and the similarity score they yield is used to determine whether two protein sequences are homologous or not. While such models are powerful, they can only model each position in a protein independently of each other, yet it is well-known that two residues that are distant in the sequence can co-evolve, for instance when they are in contact in the 3D structure. In our work, we proposed to use a different type of model: the <b>Potts model</b>, a pairwise Markov Random Field originally introduced by <a href="https://en.wikipedia.org/wiki/Direct_coupling_analysis">Direct Coupling Analysis</a> to predict contacts and protein-protein interactions. Its parameters can describe both positional conservation and couplings between positions, which make it a great candidate to model sets of homologous proteins.
                
                <br/>
								
		<img src="images/figure_msa_potts.png" alt="" width="50%" />
		<br/>
		    During my PhD, I designed <b>PPalign</b>, a method to align two Potts models and give a similarity score for this alignment. The feasibility of this alignment method was assessed on a <a href="https://www-dyliss.irisa.fr/PPalign/">benchmark</a> of low sequence identity reference sequence alignments, where suboptimal solutions were found in tractable time and the quality of our alignments was on average slightly better than our main competitor HHalign with respect to reference alignments, with better results on pairs with the lowest sequence identity <cite_publi>[1]</cite_publi>. It was presented at the JOBIM 2020 conference <cite_publi>[2]</cite_publi> during the Protein and Structure parallel session on July 3 2020, slides are available <a href="jobim_2020_compotts_hugo_talibart.pdf">here</a>.
		<br/>
		<br/>
                
                Since Potts models were originally designed to predict co-evolving positions within a given set of homologous sequences, comparing two Potts models representing two different sets of homologous sequences raised additional challenges and a need for canonicity. To address these challenges, I worked with <a href="https://isyeb.mnhn.fr/fr/annuaire/mathilde-carpentier-368">Mathilde Carpentier</a> at <a href="https://isyeb.mnhn.fr/fr/atelier-de-bio-informatique-384">Atelier de BioInformatique</a> (Institut de Systématique, Évolution, Biodiversité (ISYEB), Muséum National d'Histoire Naturelle (MNHN), Paris) as a post-doctoral researcher from 2021 to 2022. We paved the way for the modelling of protein sequences with direct coupling information for the special task of pairwise comparison.
                

		    <br/>
		    <br/>
		    Early in my thesis, I also had the opportunity to work with <a href="http://www.kotulska-lab.pwr.wroc.pl/websites/wdyrka/doku.php?id=wiki:people:witold">Witold Dyrka</a> from Politechnika Wrocławska (Wrocław, Poland) on probabilistic context-free grammars with contact map constraints on proteins <cite_publi>[3]</cite_publi>. In this work, we established a framework to train probabilistic context-free grammars representing families of protein sequences using information on contact constraints.
		    
		    <br/>
		    <br/>
		    
						Though <b>protein language processing</b> was my first love, I have become progressively interested in <b>structural bioinformatics</b>, and in the design of biologically active molecules.
			<br/>
			<br/>
						
						As of may 2023, I am a postdoctoral research at <a href="https://3bio.polytech.ulb.be/en/bioinfo">3BIO-BioInfo</a> team at Université Libre de Bruxelles, working with Dimitri Gilis on de novo <b>drug design</b> for olfactory receptors using <b>deep learning generative models</b>.
								</p>
								</div>
							</section>

							
			
							<section id="publications">
								<div class="container">
									 <h3>Publications</h3>
							                <!-- This document was automatically generated with bibtex2html 1.99
									     (see http://www.lri.fr/~filliatr/bibtex2html/),
									     with the following command:
									     bibtex2html my_biblio.bib  -->


									<table>
										
										
										<tr valign="top">
										<td align="right" class="bibtexnumber">
										[<a name="talibart2021ppalign">1</a>]
										</td>
										<td class="bibtexitem">
										Hugo Talibart and Fran&ccedil;ois Coste.
										 PPalign: optimal alignment of potts models representing proteins with
										  direct coupling information.
										 <em>BMC bioinformatics</em>, 22(1):1--22, 2021.
										[&nbsp;<a href="my_biblio_bib.html#talibart2021ppalign">bib</a>&nbsp;]

										</td>
										</tr>

									<tr valign="top">
									<td align="right" class="bibtexnumber">
									[<a name="talibart2020compotts" href="https://www.biorxiv.org/content/10.1101/2020.06.12.147702v1">2</a>]
									</td>
									<td class="bibtexitem">
									Hugo Talibart and Fran&#x00E7;ois Coste.
									 Compotts: Optimal alignment of coevolutionary models for protein
									  sequences.
									 In <em>JOBIM 2020-Journ&eacute;es Ouvertes Biologie, Informatique et
									  Math&eacute;matiques</em>, 2020.
									[&nbsp;<a href="my_biblio_bib.html#talibart2020compotts">bib</a>&nbsp;]

									</td>
									</tr>



									<tr valign="top">
									<td align="right" class="bibtexnumber">
									[<a name="dyrka2019estimating" href="https://peerj.com/articles/6559/">3</a>]
									</td>
									<td class="bibtexitem">
									Witold Dyrka, Mateusz Pyzik, Fran&#x00E7;ois Coste, and Hugo Talibart.
									 Estimating probabilistic context-free grammars for proteins using
									  contact map constraints.
									 <em>PeerJ</em>, 7:e6559, 2019.
									[&nbsp;<a href="my_biblio_bib.html#dyrka2019estimating">bib</a>&nbsp;]

									</td>
									</tr>
									</table>
								</div>
							</section>


							<section id="misc">
								<div class="container">
									<h3>Science outreach</h3>
									 <p>As part of the 2018 Sciences en Cour[t]s festival, I explained my PhD project in a short video using slips of paper. (Video in french)</p>
<div style="text-align: center;"><iframe width="560" height="315" src="https://www.youtube.com/embed/OucnrOl-xSw" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></div>
								</div>
							</section>


							<section id="bio">
								<div class="container">
									<h3>Resume</h3>
									<p> Download full resume <a href="CV_Talibart_Hugo.pdf">here</a></p>

									<h4>Experience</h4>
									<date>2023-now</date>
									<cv_item>
										<endroit>3BIO-BioInfo, Université Libre de Bruxelles, Belgique</endroit>
										<br/>
										<item_name>1-year postdoc</item_name>
										<cv_description>Artificial intelligence design of ligands for olfactory receptors</cv_description>.
									</cv_item>
									<br/>
									<date>2021-2022</date>
									<cv_item>
										<endroit>Atelier de BioInformatique, Muséum National d'Histoire Naturelle, France</endroit>
										<br/>
										<item_name>1-year postdoc</item_name>
										<cv_description>Further development on the alignment method designed during my PhD thesis. Application to homology detection.</cv_description>.
									</cv_item>
									<br/>
									<date>2021</date>
									<cv_item>
										<endroit>Dyliss Team, IRISA, Université de Rennes 1, France</endroit>
										<br/>
										<item_name>1-month engineering position</item_name>
										<cv_description>Additional development on the alignment method designed during my PhD thesis.</cv_description>.
									</cv_item>
									<br/>
									<date>2017-2021</date>
									<cv_item>
										<endroit>Dyliss Team, IRISA, Université de Rennes 1, France</endroit>
										<br/>
										<item_name>PhD thesis</item_name>
										<cv_description>Comparison of homologous protein sequences using direct coupling information by pairwise Potts model alignments. Supervised by <a href="http://people.rennes.inria.fr/Francois.Coste/">François Coste</a> and <a href="http://people.rennes.inria.fr/Jacques.Nicolas/">Jacques Nicolas</a></cv_description>.
									</cv_item>
									<br/>
									<date>2017</date>
									<cv_item>
										<endroit>LITIS Lab, Université de Rouen, France</endroit>
										<br/>
										<item_name>6-month internship</item_name>
										<cv_description>Supervised learning of emergent structures in agent-based simulations. Supervised by <a href="http://pagesperso.litislab.fr/ptranouez/">Pierrick Tranouez</a>.</cv_description>
									</cv_item>
									<br/>
									<date>2016</date>
									<cv_item>
										<endroit>King's College London, United Kingdom</endroit>
										<br/>
										<item_name>3-month internship</item_name>
										<cv_description>RNA meta-stable secondary structures clustering. Supervised by <a href="https://www.kcl.ac.uk/people/kathleen-steinhofel">Kathleen Steinhöfel</a>.</cv_description>

									<br/>
									<br/>

									<h4>Education</h4>

									<date>2016-2017</date>
									<cv_item>
									    <endroit>Université de Rouen, France</endroit>
									    <br/>
									    <item_name>Master's Degree in Theoretical Computer Science</item_name>
									    <cv_description>Automata theory, String Algorithms in Bioinformatics</cv_description>
									 </cv_item>
									   <br/>
									 <date>2012-2017</date>
									    <cv_item>
									    <endroit>Institut National des Sciences Appliquées de Rouen (INSA Rouen), France</endroit>
									    <br/>
									    <item_name>Engineering Degree in Mathematics</item_name>
									    <cv_description>Statistics, Artificial Intelligence, Operations Research</cv_description>
									</cv_item>
								</div>


							</section>



							<section id="contact">
								<div class="container">
									<h3>Contact</h3>
								<p>
								    E-mail: 
									 
									 <script language="JavaScript">
									var username = "hugo.talibart";
									var hostname = "protonmail.com";
									var linktext = username + "@" + hostname ;
									document.write("<a href='" + "mail" + "to:" + username + "@" + hostname + "'>" + linktext + "</a>");
									</script>
								</p>

								</div>
							</section>


						
					
					</div>

				<!-- Footer -->
					<section id="footer">
						<div class="container">
							<ul class="copyright">
								<li>Design made using: <a href="https://html5up.net/read-only">HTML5 UP</a></li>
							</ul>
						</div>
					</section>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
