<!DOCTYPE HTML>
<!--
	Read Only by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Hugo Talibart | Bioinformatician</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>
	<body class="is-preload">

		<!-- Header -->
			<section id="header">
				<annonce>Available for a postdoc or engineer position <br/> as of 09/2022</annonce>
				<header>
					<span class="image avatar"><img src="images/avatar.jpg" alt="" /></span>
					<h1 id="logo"><a href="index.html">Hugo Talibart</a></h1>
					<p>
						Postdoctoral researcher <br/>
						<a href="https://isyeb.mnhn.fr/fr/atelier-de-bio-informatique-384">Atelier de BioInformatique</a> <br/>
						Muséum National d'Histoire Naturelle (MNHN) <br/>
						Paris, France
						<br/>
						<br/>
					</p>
				</header>
				<nav id="nav">
					<ul>
						<li><a href="#overview" class="active">overview</a></li>
						<li><a href="#publications">publications</a></li>
						<li><a href="#misc">misc.</a></li>
						<li><a href="#bio">resume</a></li>
						<li><a href="#contact">contact</a></li>
					</ul>
				</nav>
				<footer>
					<ul class="icons">
						<li><a href="https://twitter.com/HTalibart" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
						<!--<li><a href="#" class="icon brands fa-github"><span class="label">Github</span></a></li>-->
						<li><a href="mailto:hugo.talibart@irisa.fr" class="icon solid fa-envelope"><span class="label">Email</span></a></li>
					</ul>
				</footer>
			</section>

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Main -->
					<div id="main">
							<section id="overview">
								<div class="image main" data-position="center">
									<img src="images/banner.jpg" alt="" />
								</div>
								<div class="container">
									<header class="major">
										<h3>Overview</h3>
									</header>
		<p>
			As a nascent bioinformatician, my ambition is to use my initial background in computer science and applied mathematics to
				help expand knowledge on the living world. I am more specifically interested in proteins - the building blocks of life -
				their evolution, classification into families and structures. <br/>
			
			From 2017 to early 2021 I carried out a PhD at the University of Rennes 1, in <a href="http://www.irisa.fr/dyliss"/>Dyliss</a> team,
                under the supervision of <a href="http://people.rennes.inria.fr/Francois.Coste/">François Coste</a>
                and <a href="http://people.rennes.inria.fr/Jacques.Nicolas/">Jacques Nicolas</a>,
			entitled "Comparison of homologous protein sequences using direct coupling information by pairwise Potts model alignments".
                    <br/>
                    <br/>

		    This thesis focused on <b>protein homology search</b> and <b>protein family modelling</b>. As of today, the standard way of modelling a set of protein sequences is to use profile Hidden Markov Models. These statistical models represent protein families with positional information and insertion and deletion states. Widely used packages such as HH-suite provide tools to perform sequence-HMM alignment and HMM-HMM alignment and the similarity score they yield is used to determine whether two protein sequences are homologous or not. While powerful, HMMs are limited by their positional nature. Yet, it is well-known that two residues that are distant in the sequence can co-evolve, for instance because they are in contact in the 3D structure. In our work, we propose to use a different type of model: the <b>Potts model</b>. This model was introduced by <a href="https://en.wikipedia.org/wiki/Direct_coupling_analysis">Direct Coupling Analysis</a>, originally to predict contacts and protein-protein interactions. Its parameters can describe both positional conservation and couplings between positions, which make it a great candidate to model sets of homologous proteins.
		    <br/>
		    <br/>
		    During my PhD, I designed <b>PPalign</b>, a method to align two Potts models and give a similarity score for this alignment. The feasibility of this alignment method was assessed on a <a href="https://www-dyliss.irisa.fr/PPalign/">benchmark</a> of low sequence identity reference sequence alignments, where suboptimal solutions were found in tractable time and the quality of our alignments was on average slightly better than our main competitor HHalign with respect to reference alignments, with better results on pairs with the lowest sequence identity <cite_publi>[2]</cite_publi>.
		    <br/>
		    <br/>
		    I presented my method at the conference JOBIM 2020 <cite_publi>[1]</cite_publi> during the Protein and Structure parallel session on July 3 2020, my slides are available <a href="jobim_2020_compotts_hugo_talibart.pdf">here</a>.
		    <br/>
		    <br/>
		    I am currently working as a postdoctoral researcher at the <a href="https://isyeb.mnhn.fr/fr/atelier-de-bio-informatique-384">Atelier de BioInformatique</a> at the Institut de Systématique, Évolution, Biodiversité (ISYEB) at the Muséum National d'Histoire Naturelle (MNHN Paris)
								with <a href="https://isyeb.mnhn.fr/fr/annuaire/mathilde-carpentier-368">Mathilde Carpentier</a>.
		    <br/>
		    <br/>
		    Early in my thesis, I also had the opportunity to work with <a href="http://www.kotulska-lab.pwr.wroc.pl/websites/wdyrka/doku.php?id=wiki:people:witold">Witold Dyrka</a> from Politechnika Wrocławska (Wrocław, Poland) on probabilistic context-free grammars with contact map constraints on proteins <cite_publi>[3]</cite_publi>. In this work, we established a framework to train probabilistic context-free grammars representing families of protein sequences using information on contact constraints.

								</p>
								</div>
							</section>

							
			
							<section id="publications">
								<div class="container">
									 <h3>Publications</h3>
							                <!-- This document was automatically generated with bibtex2html 1.99
									     (see http://www.lri.fr/~filliatr/bibtex2html/),
									     with the following command:
									     bibtex2html my_biblio.bib  -->


									<table>

									<tr valign="top">
									<td align="right" class="bibtexnumber">
									[<a name="talibart2020compotts" href="https://www.biorxiv.org/content/10.1101/2020.06.12.147702v1">1</a>]
									</td>
									<td class="bibtexitem">
									Hugo Talibart and Fran&#x00E7;ois Coste.
									 Compotts: Optimal alignment of coevolutionary models for protein
									  sequences.
									 In <em>JOBIM 2020-Journ&eacute;es Ouvertes Biologie, Informatique et
									  Math&eacute;matiques</em>, 2020.
									[&nbsp;<a href="my_biblio_bib.html#talibart2020compotts">bib</a>&nbsp;]

									</td>
									</tr>

									<tr valign="top">
									<td align="right" class="bibtexnumber">
									[<a name="talibart2020ppalign">2</a>]
									</td>
									<td class="bibtexitem">
									Hugo Talibart and Fran&#x00E7;ois Coste.
									 Ppalign: Optimal alignment of potts models representing proteins with
									  direct coupling information.
									 <em>bioRxiv</em>, 2020.
									[&nbsp;<a href="my_biblio_bib.html#talibart2020ppalign">bib</a>&nbsp;]

									</td>
									</tr>


									<tr valign="top">
									<td align="right" class="bibtexnumber">
									[<a name="dyrka2019estimating" href="https://peerj.com/articles/6559/">3</a>]
									</td>
									<td class="bibtexitem">
									Witold Dyrka, Mateusz Pyzik, Fran&#x00E7;ois Coste, and Hugo Talibart.
									 Estimating probabilistic context-free grammars for proteins using
									  contact map constraints.
									 <em>PeerJ</em>, 7:e6559, 2019.
									[&nbsp;<a href="my_biblio_bib.html#dyrka2019estimating">bib</a>&nbsp;]

									</td>
									</tr>
									</table>
								</div>
							</section>


							<section id="misc">
								<div class="container">
									<h3>Science outreach</h3>
									 <p>As part of the 2018 Sciences en Cour[t]s festival, I explained my PhD project in a short video using slips of paper. (Video in french)</p>
<div style="text-align: center;"><iframe width="560" height="315" src="https://www.youtube.com/embed/OucnrOl-xSw" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></div>
								</div>
							</section>


							<section id="bio">
								<div class="container">
									<h3>Resume</h3>
									<p> Download full resume <a href="CV_Talibart_Hugo.pdf">here</a></p>

									<h4>Experience</h4>
									<date>2017-now</date>
									<cv_item>
										<endroit>Dyliss Team, IRISA, Université de Rennes 1, France</endroit>
										<br/>
										<item_name>PhD thesis</item_name>
										<cv_description>Comparison of homologous protein sequences using direct coupling information by pairwise Potts model alignments. Supervised by <a href="http://people.rennes.inria.fr/Francois.Coste/">François Coste</a> and <a href="http://people.rennes.inria.fr/Jacques.Nicolas/">Jacques Nicolas</a></cv_description>.
									</cv_item>
									<br/>
									<date>2017</date>
									<cv_item>
										<endroit>LITIS Lab, Université de Rouen, France</endroit>
										<br/>
										<item_name>6-month internship</item_name>
										<cv_description>Supervised learning of emergent structures in agent-based simulations. Supervised by <a href="http://pagesperso.litislab.fr/ptranouez/">Pierrick Tranouez</a>.</cv_description>
									</cv_item>
									<br/>
									<date>2016</date>
									<cv_item>
										<endroit>King's College London, United Kingdom</endroit>
										<br/>
										<item_name>3-month internship</item_name>
										<cv_description>RNA meta-stable secondary structures clustering. Supervised by <a href="https://www.kcl.ac.uk/people/kathleen-steinhofel">Kathleen Steinhöfel</a>.</cv_description>

									<br/>
									<br/>

									<h4>Education</h4>

									<date>2016-2017</date>
									<cv_item>
									    <endroit>Université de Rouen, France</endroit>
									    <br/>
									    <item_name>Master's Degree in Theoretical Computer Science</item_name>
									    <cv_description>Automata theory, String Algorithms in Bioinformatics</cv_description>
									 </cv_item>
									   <br/>
									 <date>2012-2017</date>
									    <cv_item>
									    <endroit>Institut National des Sciences Appliquées de Rouen (INSA Rouen), France</endroit>
									    <br/>
									    <item_name>Engineering Degree in Mathematics</item_name>
									    <cv_description>Statistics, Artificial Intelligence, Operations Research</cv_description>
									</cv_item>
								</div>


							</section>



							<section id="contact">
								<div class="container">
									<h3>Contact</h3>
								<p>
								    Dyliss team, Irisa / Inria Rennes-Bretagne Atlantique, <br/>
								    Campus de Beaulieu, 35042 Rennes Cedex, France. <br/>
								    Tel: +33 (0) 2 99 84 22 32 <br/>
								    E-mail: 
									 
									 <script language="JavaScript">
									var username = "hugo.talibart";
									var hostname = "irisa.fr";
									var linktext = username + "@" + hostname ;
									document.write("<a href='" + "mail" + "to:" + username + "@" + hostname + "'>" + linktext + "</a>");
									</script>
								</p>

								</div>
							</section>


						
					
					</div>

				<!-- Footer -->
					<section id="footer">
						<div class="container">
							<ul class="copyright">
								<li>Design adapted from: <a href="https://html5up.net/read-only">HTML5 UP</a></li>
							</ul>
						</div>
					</section>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
